build_compilers/test/modules_test.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:analyzer/dart/element/element.dart';
+import 'package:build_test/build_test.dart';
+import 'package:test/test.dart';
+
+import 'package:build_compilers/src/modules.dart';
+
+void main() {
+  LibraryElement libCycle;
+  LibraryElement libSecondaryInCycle;
+  LibraryElement libNoCycle;
+  LibraryElement libCycleWithB;
+  LibraryElement libCycleWithA;
+  final assetCycle = makeAssetId('a|lib/a_cycle.dart');
```

Comment:
nit: rename `assetCycleA` (and `libCycleA` etc)?

Reply:
The interesting bit is that it's a cycle, not that it's in package A. Would prefer to keep this as is unless you feel strongly about it.

Reply:
It just makes it harder to understand the test imo, but I don't feel that strongly. I found myself jumping around this file a lot when reviewing it to understand what was going on etc.